### PR TITLE
timidity: Add new package

### PR DESCRIPTION
**Pull request by hliebel**
Midi player for game/openttd

### CODE REVIEW

**Comment by alarcher** [comment]
The package gets an unresolved dependency on libao if it is installed on the system. You should either enable the option and update REQUIRED_PACKAGES, or disable it explicitly.

**Comment by alarcher** [comment]
@hliebel Sorry for the delay, everyone seems pretty busy lately.

File: components/multimedia/timidity++/patches/01-sys-soundcard.patch [hunk 1]
```diff
@@ -0,0 +1,25 @@
+diff -puNr TiMidity++-2.14.0.orig/timidity/midi_a.c TiMidity++-2.14.0/timidity/midi_a.c
+--- TiMidity++-2.14.0.orig/timidity/midi_a.c	2009-03-14 15:49:59.000000000 +0000
++++ TiMidity++-2.14.0/timidity/midi_a.c	2018-04-10 07:35:49.743687930 +0000
+@@ -49,14 +49,14 @@
+ #endif
+ #include <fcntl.h>
+ #include <stdlib.h>
+-#ifdef HAVE_SYS_SOUNDCARD_H
```
Comment (alarcher) [inline components/multimedia/timidity++/patches/01-sys-soundcard.patch]: Maybe guard it with a conditional on __sun to make it cleaner.

**Comment by alarcher** [comment]
Depends on #4077

**Comment by hliebel** [comment]
I have 'library/audio/libao' in the Makefile REQUIRED_PACKAGES already. When I do a 'gmake env-check' the library is listed as required. When I do a ldd on the binary it refers to the library provided by that package. If I loook at the raw manifest for this package it lists the libao library as a depend.
Not sure what to look at otherwise..?

**Comment by alarcher** [comment]
Nevermind... I had library/libao installed from before I renamed it and added the package to the repository...

**Comment by alarcher** [comment]
Did not see that there were 2 commits... sorry...

**Pull request merged**